lib/visitor.js

```diff
@@ -336,9 +336,12 @@ class Visitor {
     /**
      * @param {string} fq - fully qualified name of the entity
      * @param {string} clean - the clean name of the entity
+     * @param {boolean} [isPlural] - `true` if passed entity is plural
      */
-    #staticClassContents(fq, clean) {
-        return isDraftEnabled(fq) ? [`static drafts: typeof ${clean}`] : []
+    #staticClassContents(fq, clean, isPlural = false) {
+        if (!isDraftEnabled(fq)) return []
+        const draftsType = isPlural ? '__.DraftsOf' : '__.DraftOf'
```

Comment:
Please add respective [wrappers](https://github.com/cap-js/cds-typer/blob/main/lib/components/wrappers.js) for these two code fragments for consistency